### PR TITLE
Enables Novel Posting

### PR DESCRIPTION
**Pull request by R3dtail**
## About The Pull Request

Doubles the emote limit from 2048 to 4096.

## Why It's Good For The Game

  Emoting is good, increasing the ability of people to write to their hearts content is also good.  

## Changelog
:cl:

tweak: tweaked a few things
code: changed some code
/:cl:

### CODE REVIEW

File: code/__DEFINES/say.dm [hunk 1]
```diff
@@ -86,7 +86,7 @@
 #define EMOTE_OMNI 4
 
 //Don't set this very much higher then 1024 unless you like inviting people in to dos your server with message spam
-#define MAX_MESSAGE_LEN			2048		//Citadel edit: What's the WORST that could happen?
+#define MAX_MESSAGE_LEN			4096		//Citadel edit: What's the WORST that could happen?
 #define MAX_FLAVOR_LEN			4096		//double the maximum message length.
```
Comment (timothyteakettle) [inline code/__DEFINES/say.dm]: just remove the comment here as it's no longer true

**Comment by timothyteakettle** [comment]
changelog is supposed to be something like tweak: doubled maximum emote length, also see my other comment

**Comment by dzahlus** [comment]
fetish PR :^)

**Comment by keronshb** [comment]
Time to flex on ghosts with /me's

**Pull request merged**